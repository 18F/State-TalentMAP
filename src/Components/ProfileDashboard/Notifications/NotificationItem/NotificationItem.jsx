import React from 'react';
import PropTypes from 'prop-types';
import InformationDataPoint from '../../InformationDataPoint';

const NotificationItem = ({ content, notificationTime }) => (
  <div className="usa-grid-full notification-card">
    <InformationDataPoint content={content} title={notificationTime} titleOnBottom />
  </div>
);

NotificationItem.propTypes = {
  content: PropTypes.node.isRequired,
  notificationTime: PropTypes.string.isRequired,
};

export default NotificationItem;
