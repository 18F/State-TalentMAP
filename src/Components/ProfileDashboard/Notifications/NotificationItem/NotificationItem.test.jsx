import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import NotificationItem from './NotificationItem';

describe('NotificationItemComponent', () => {
  it('is defined', () => {
    const wrapper = shallow(
      <NotificationItem
        content="content"
        notificationTime="10-10-2017"
      />,
    );
    expect(wrapper).toBeDefined();
  });

  it('matches snapshot when editor is shown', () => {
    const wrapper = shallow(
      <NotificationItem
        content="content"
        notificationTime="10-10-2017"
      />,
    );
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
