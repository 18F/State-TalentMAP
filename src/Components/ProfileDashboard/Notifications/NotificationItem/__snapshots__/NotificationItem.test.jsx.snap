// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotificationItemComponent matches snapshot 1`] = `
<div
  className="usa-grid-full notification-card"
>
  <InformationDataPoint
    className=""
    content="content"
    title="about 1 month ago"
    titleOnBottom={true}
  />
</div>
`;
